packages/core/src/locale/language-config.ts

```diff
@@ -6,12 +6,28 @@ export default interface LanguageConfig {
     locale: string;
     locales: Locales;
     translations: Translations;
+    /** @hidden */
```

Comment:
Just to confirm, is this comment for `typedoc`?

Reply:
Yes, I will add a comment there to make it clearer. 🙂